src/ethereum/berlin/spec.py

```diff
@@ -1002,10 +1002,11 @@ def calculate_block_difficulty(
     # bomb has no effect prior to block 200000 we pretend it existed from
     # genesis.
     # See https://github.com/ethereum/go-ethereum/pull/1588
-    num_bomb_periods = ((int(block_number) - BOMB_DELAY_BLOCKS) // 100000) - 2
+    num_bomb_periods = (int(block_number) - BOMB_DELAY_BLOCKS) // 100000 - 2
```

Comment:
I know there's no ambiguity with the new version, but I'd prefer to keep the extra parentheses.